source/Halibut.TestUtils.CompatBinary.Base/ProxyServiceForwardingRequestToClient.cs

```diff
@@ -113,5 +110,32 @@ public static async Task Run()
                 }
             }
         }
+
+        public static int WebSocketListeningPort(HalibutRuntime client, out string webSocketListeningUrl)
```

Comment:
Can you not use `TryListenWebSocket` here? I did the add an existing file as a link for `TcpPortHelper` as a very lazy shared library!!

Reply:
Ahh it is a bit different!!